rest-api/config/config_test_ptsc_1.json

```diff
@@ -0,0 +1,45 @@
+{
+  "allow_nonprod_requests": true,
+  "biobank_id_prefix": ["Y"],
```

Comment:
Invent a new biobank prefix for each environment?

@Seeker1911

Reply:
I think just use the test prefix here, so PTSC doesn't need to adjust configs.